secure_message/common/utilities.py

```diff
@@ -70,19 +68,19 @@ def paginated_list_to_json(paginated_list, page, limit, host_url, user, string_q
     for message in paginated_list.items:
         msg_count += 1
         msg = message.serialize(user, body_summary=True)
-        msg['_links'] = {"self": {"href": "{0}{1}/{2}".format(host_url, MESSAGE_BY_ID_ENDPOINT, msg['msg_id'])}}
+        msg['_links'] = {"self": {"href": f"{host_url}{MESSAGE_BY_ID_ENDPOINT}/{msg['msg_id']}"}}
         messages.append(msg)
 
-    links = {'first': {"href": "{0}{1}".format(host_url, endpoint)},
-             'self': {"href": "{0}{1}{2}{3}page={4}&limit={5}".format(host_url, endpoint, arg_joiner, string_query_args, page, limit)}}
+    links = {'first': {"href": f"{host_url}{endpoint}"},
+             'self': {"href": f"{host_url}{endpoint}{arg_joiner}{string_query_args}page={page}&limit={limit}"}}
 
     if paginated_list.has_next:
         links['next'] = {
-            "href": "{0}{1}{2}{3}page={4}&limit={5}".format(host_url, endpoint, arg_joiner, string_query_args, (page + 1), limit)}
+            "href": f"{host_url}{endpoint}{arg_joiner}{string_query_args}page={page + 1}&limit={limit}"}
 
     if paginated_list.has_prev:
         links['prev'] = {
-            "href": "{0}{1}{2}{3}page={4}&limit={5}".format(host_url, endpoint, arg_joiner, string_query_args, (page - 1), limit)}
+            "href": f"{host_url}{endpoint}{arg_joiner}{string_query_args}page={page - 1}&limit={limit}"}
```

Comment:
So much tidier with f string formatting. I think we should be using f strings across the board